microsetta_public_api/api/microsetta_public_api.yml

```diff
@@ -1580,6 +1645,40 @@ components:
             - 0.05
             - 0.11
 
+    taxonomyRanks:
```

Comment:
Before we stick with this schema, do you think it will be relatively easy to consume in plotly?

Reply:
@dhakim87

Reply:
Isn't Taxa-order implied by Taxon and Rank?

I'm not sure Plotly has a pure rank plot built in, closest I've seen is probably the racing bar chart, which would be great if we had some additional metric to show alongside each taxon, like relative abundance or something, especially if we had longitudinal data.  

See https://www.youtube.com/watch?v=tIyguSwqObA&feature=emb_logo for this kind of longitudinal plot.

Reply:
It is not as this is a random sample from a set of samples. So you could have Taxon ['a', 'b', 'b', 'c', 'd'], and Rank [1, 2, 3, 4, 5]

Ideally we replicate the proof of concept taxa rank plot. For matplotlib/seaborn, this is passing in the Taxons as "y", Rank as "x" and the Taxa-order as the y-ordering

![Screen Shot 2020-12-10 at 1 43 39 PM](https://user-images.githubusercontent.com/474290/101833384-bdbef280-3aed-11eb-9baa-84b7727ea302.png)